sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client_options.hpp

```diff
@@ -126,7 +127,7 @@ namespace Azure { namespace Security { namespace Attestation {
   {
     /** @brief Version to use when communicating with the attestation service.
      */
-    ServiceVersion Version;
+    ServiceVersion Version{ServiceVersion::V2020_10_01};
```

Comment:
API Review mentioned that the value wasn't discoverable - this might be a limitation of the current ApiView implementation but I got explicit feedback to add a default constructor so it can be discoverable.

Reply:
It should be enough to add a comment to the field to say that the value is set with the constructor.
Otherwise, we can have a sync bug in the future where we believe we updated the apiVersion by changing the default value for the field, but the constructor would still be setting the old value.

Reply:
Or, you could also remove the parameters from the constructor.  Let the constructor only call base-class constructor but let the field's default values.  It would be different from the other SDK clients (storage, key vault) but I think it makes sense

Reply:
I went back to the ApiReview feedback, I think this is primarily and ApiView issue and not worth fixing.